components/telegram_bot_api/telegram_bot_api.app.js

```diff
@@ -1,4 +1,20 @@
+// If unset, set `process.env.NTBA_FIX_319` to true to enable cancellation of
+// Promises. See: https://github.com/yagop/node-telegram-bot-api/issues/319 for
+// more information.
+process.env.NTBA_FIX_319 = (process.env.NTBA_FIX_319 !== undefined)
+  ? process.env.NTBA_FIX_319
+  : true;
+const TelegramBot = require("node-telegram-bot-api");
 const axios = require("axios");
+const {
+  TELEGRAM_BOT_API_UI_MEDIA_AUDIO,
+  TELEGRAM_BOT_API_UI_MEDIA_DOCUMENT,
+  TELEGRAM_BOT_API_UI_MEDIA_PHOTO,
+  TELEGRAM_BOT_API_UI_MEDIA_VIDEO,
+  TELEGRAM_BOT_API_UI_MEDIA_VIDEO_NOTE,
+  TELEGRAM_BOT_API_UI_MEDIA_STICKER,
+  TELEGRAM_BOT_API_UI_MEDIA_VOICE,
+} = require("./constants");
 const updateTypes = [
```

Comment:
My suggesetion here is to set this `const` as a constant variable in `constants` file

Reply:
I created a file with this `const` called `update-types.mjs`, similar to [twist/event-types.js](https://github.com/PipedreamHQ/pipedream/blob/master/components/twist/event-types.js) or [todoist/resource-types.js](https://github.com/PipedreamHQ/pipedream/blob/master/components/todoist/resource-types.js). Is that okay, or would you suggest putting `updateTypes` in `constants.mjs`?